src/lang/po/fr/Prusa-Firmware-Buddy_fr.po

```diff
@@ -3542,7 +3542,7 @@ msgstr "Aucune configuration"
 
 #: src/gui/MItem_tools.cpp:287
 msgid "No crash dump to save."
-msgstr "Aucun dump de crash à sauvegarder."
+msgstr "Aucun rapport de vidage de mémoire à sauvegarder."
```

Comment:
A "rapport" is a report, in this case, it is a crash dump, thus the translation is correct.

Reply:
fixed (reverted)

Reply:
Please see my other comment; crash dump is not related to "vidage de mémoire".